source/stylesheets/refills/unstyled/_switch.scss

```diff
@@ -0,0 +1,50 @@
+.switch {  
+  $switch-width: 52px;
+  $switch-padding: 2px;
+  $switch-height: 32px;
+  $knob-size: $switch-height - ($switch-padding * 2);
+  $knob-width: $knob-size;
```

Comment:
These variables should go above Line 1, `.switch`, outside the declaration block.